src/utilities/forms.py

```diff
@@ -87,3 +91,16 @@ def clean(self, data, initial=None):
         #         pass
 
 # http://k
+
+
+class CustomFlatpageForm(FlatpageForm):
+    content = SummernoteTextField()
```

Comment:
`content = SummernoteTextField()`
This line isn't necessary I'm pretty sure.  Is this used any where else in this project?  The default field is fine, we just need the widget (unless you've got a specific reason to use this?  Is it from the latest documentation maybe?)

Reply:
I got it from django-summernote github: https://github.com/summernote/django-summernote

in the read-me referencing SummernoteTextField():
![image](https://user-images.githubusercontent.com/39788517/172508969-4a55c890-e52d-4ca8-9ee7-0ca546120ddb.png)

Since I'm not sure if we already have code dealing with it I just added it

Reply:
Oh this is good stuff.  Please remove it for now (so it matches our other uses of Summernote in the app) and I added a new issue to look into this, #1012. Thank you!

Reply:
Removed from flatpage form